### PR TITLE
Update attribute used for AAP from AWS marketplace

### PR DESCRIPTION
**Pull request by ianf77**
Updated attribute and did some tidying up.

### CODE REVIEW

File: stories/topics/proc-aap-aws-traffic-to-vpc.adoc [hunk 1]
```diff
@@ -11,7 +11,7 @@
 You can set other connectivity types, such as ICMP for pings to the EC2 instances deployed with the solution.
 
 . Set *Source type* to *Custom*.
-. In the *Source* field, enter the CIDR block of the {PlatformNameShort} from {Market} VPC.
+. In the *Source* field, enter the CIDR block of the {AAPonAWS} VPC.
 . Click btn:[Save rule].
 
-If you have configured more restrictive outbound rules, then can repeat the previous steps for outbound rules to ensure that your existing VPC can communicate to {PlatformNameShort} from {Market} for UI and API access.
+If you have configured more restrictive outbound rules, then can repeat the previous steps for outbound rules to ensure that your existing VPC can communicate to {AAPonAWS} for UI and API access.
```
Comment (ariordan-redhat) [inline stories/topics/proc-aap-aws-traffic-to-vpc.adoc:17]:
```suggestion
If you have configured more restrictive outbound rules, then you can repeat the previous steps for outbound rules to ensure that your existing VPC can communicate to {AAPonAWS} for UI and API access.
```

File: stories/topics/proc-aap-aws-uninstall.adoc [hunk 2]
```diff
@@ -26,12 +27,13 @@ If you did not install extension nodes, skip theses steps.
 .. Confirm the deletion.
 . Extension nodes have been removed
 
-== Uninstall {AAPonAWS}
+== Uninstalling {AAPonAWS}
 
 .Procedure
 . In the AWS UI, navigate to the link:https://us-east-1.console.aws.amazon.com/cloudformation/home?region=us-east-1[cloudformation page].
```
Comment (ariordan-redhat) [inline stories/topics/proc-aap-aws-uninstall.adoc:33]: Should this URL be a bit more generic - ie not specify a region?

File: stories/topics/ref-aap-aws-additional-configs-update-rds-password.adoc [hunk 3]
```diff
@@ -41,10 +42,10 @@ $ ALTER USER awx WITH PASSWORD <UPDATED_PASSWORD>;
 The autoscale group restarts them.
 .. Wait for the {ControllerName} Auto Scale group and the {HubName} Auto Scaling group to create new EC2 instances.
 .. The new RDS can be used when the new {ControllerName} and {HubName} EC2 instances reach a _Running_ Instance State.
-.. To validate the RDS password has been changed -
-... Identify and connect to the {HubName} instance for the CloudFormation Stack.
-... View the updated database settings from the following file - `/etc/pulp/settings.py`.
-... Attempt to connecct using the following sample connection string:
+. To validate the RDS password has been changed -
```
Comment (ariordan-redhat) [inline stories/topics/ref-aap-aws-additional-configs-update-rds-password.adoc:45]:
```suggestion
. To validate the RDS password has been changed:
```

**Review by ariordan-redhat — COMMENTED** [review]
A couple of suggestions. LGTM otherwise.

File: stories/topics/proc-aap-aws-uninstall.adoc (hunk 2 above)
Comment (ianf77) [reply stories/topics/proc-aap-aws-uninstall.adoc:33]: I think this is the region it's all set up on.  I've seen it mentioned lots of times in the AWS Slack channel.

**Review by ariordan-redhat — APPROVED** [review]
LGTM

**Pull request merged**